R/use_import_from.R

```diff
@@ -0,0 +1,69 @@
+#' Import a function from another package
+#'
+#' `use_import_from()` imports a function from another package by adding the
+#' Roxygen tag `@importFrom` to the package-level documentation (possibly
+#' created with [`use_package_doc()`]). Importing a function from another
+#' package allows you to refer to it without a namespace (e.g., `fun()` instead
+#' of `package::fun()`). Use such imports judiciously, as they make it hard to
```

Comment:
Should mention that it re-loads the package so the newly imported function is immediately available.